### DISCUSSION TITLE
Widget Card Colour

### DISCUSSION THREAD

**Discussion by afdhal395** [filamentphp/filament#6049] [Help] [answered]
Hi Everyone,

I am trying to make colorized card widget like the example picture below. I tried with ->color('warning') but that seems to inherit from ->description() method. [#2270](https://github.com/filamentphp/filament/issues/2270)

![image](https://user-images.githubusercontent.com/42729942/228440992-700d0043-c4bb-44aa-9db6-85adafc8b92d.png)

Anyone have achieve the same result as I expected?

Thank you for your time.

**Comment by danharrin** [comment, marked as answer]
Background colors arent supported in our core stats widget yet. You could build a custom widget with Livewire for this.
